config/routes.rb

```diff
@@ -27,11 +27,12 @@
     post 'update', on: :collection, as: 'update'
   end
   resources :team_types
+  resources :relationship_types
+  devise_for :users, controllers: { omniauth_callbacks: 'omniauth', registrations: 'users/registrations' }
```

Comment:
devise docs advise putting the custom `users` routes below the `devise_for :users`